text/src/swam/witx/parser/ImportContext.scala

```diff
@@ -0,0 +1,20 @@
+package swam.witx.parser
+
+import swam.witx.unresolved.BaseWitxType
+
+/**
+  *@author Javier Cabrera-Arteaga on 2020-03-18
+  */
+class ImportContext(val context: Map[String, BaseWitxType]) {
+
+  val types = context
+  def load(path: String) = {}
+
+  def getType(name: String): Unit = types(name)
```

Comment:
`Unit`? Plus the `apply` method on Map is not safe, because will throw an exception if the element is not found.